### PR TITLE
Add Project Setup Files

### PR DESCRIPTION
**Pull request by dosama**
This pull request include the following points covered :
1-	React Project Setup
2-	Json-server setup
3-	Integrating React Project with Typescript.
4-	Integrating React Project with Bootstrap 
5-	Configuring Using the Pre-Processors SASS and SCSS..
6-	Setup App Routings 
You will find in Home Page buttons to test App Routing to the following Component [Drug List – Add Drug – Edit Drugs]

### CODE REVIEW

File: integrant-react-01/images.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module '*.svg'
+declare module '*.png'
+declare module '*.jpg'
```
Comment (Petersobhy) [inline integrant-react-01/images.d.ts]: What this file is used for?

File: integrant-react-01/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "integrant-react-01",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react-router-dom": "^4.2.6",
+    "bootstrap": "^4.1.0",
+    "node-sass-chokidar": "^1.2.2",
+    "npm-run-all": "^4.1.2",
```
Comment (Petersobhy) [inline integrant-react-01/package.json]: Why do you need this package?

File: integrant-react-01/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "integrant-react-01",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react-router-dom": "^4.2.6",
+    "bootstrap": "^4.1.0",
+    "node-sass-chokidar": "^1.2.2",
```
Comment (Petersobhy) [inline integrant-react-01/package.json]: This should be dev dependencies, right?

File: integrant-react-01/src/server/db.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    "people":[{
+        "id":0,
+        "name":"Ahmad"
+    },
+    {
+        "id":1,
+        "name":"Mohamed"
+    }
+]
+}
```
Comment (Petersobhy) [inline integrant-react-01/src/server/db.json]: Shouldn't the server folder be outside the React app folder?

**Comment by Petersobhy** [comment]
Looks good ,, please install Prettier VS Code plugin and make sure to run it

File: integrant-react-01/package.json (hunk 3 above)
Comment (dosama) [reply integrant-react-01/package.json]: yes it can be transferred to dev

File: integrant-react-01/src/server/db.json (hunk 4 above)
Comment (dosama) [reply integrant-react-01/src/server/db.json]: No it is not mandatory to be outside react folder .. i hosted it inside react folder as data is related to the project .. if you need me to separate it out the project confirm please

File: integrant-react-01/images.d.ts (hunk 1 above)
Comment (dosama) [reply integrant-react-01/images.d.ts]: it is auto-generated while installing the typescript with react

Comment (Petersobhy) [reply integrant-react-01/images.d.ts]: What happens if we delete it? if not needed let's remove it

File: integrant-react-01/src/server/db.json (hunk 4 above)
Comment (Petersobhy) [reply integrant-react-01/src/server/db.json]: Yes please, let's move it outside the React app directory

File: integrant-react-01/images.d.ts (hunk 1 above)
Comment (dosama) [reply integrant-react-01/images.d.ts]: Okay

**Pull request merged**